llvm/lib/AsmParser/LLLexer.cpp

```diff
@@ -1005,6 +1009,8 @@ lltok::Kind LLLexer::Lex0x() {
     ++CurPtr;
 
   if (Kind == 'J') {
+    // FIXME: Need to add code here for DFP, using the APDecimalFloat.
+
```

Comment:
```suggestion
```
Per the C23 draft, section 6.4.4.2, Floating constants, p2, hexadecimal floating constants are not supported for DFP types. I think this comment can be removed.